docs/docs/tutorial/chapter0/what-is-redwood.md

```diff
@@ -64,6 +64,10 @@ You can also prerender pages that contain variables pulled from the URL, like th
 
 This is Redwood's version of static site generation, aka SSG.
 
+
+#### Accessibility
+
+Redwood includes a couple of components to [aid screen readers](https://redwoodjs.com/docs/accessibility) in properly navigating your app. The `<RouteAnnouncement>` component tells a screen reader to read something aloud, even though it isn't visible in the browser. And the `<RouteFocus>` tells a reader to skip verbose navigation options at the top of a page and get to the content.
```

Comment:
Moved "**Accessibility**" into the "**_Router_**" section, from below, because the transition sentence from the previous paragraph does not make sense leading into "_**Accessibility**_." See red line. Moving it up to the router section made the most sense out of the options I saw.
![image](https://github.com/user-attachments/assets/012c85d9-bd3b-43ae-a606-0a5517992755)